### ISSUE TITLE
TypeError: info.licenses.map is not a function

### ISSUE THREAD

**Issue by erikhofer** [sebastianroming/license-report-generator#14]
I'm trying to use the CLI but get the following error.

```
> license-report-generator

(node:14476) UnhandledPromiseRejectionWarning: TypeError: info.licenses.map is not a function
    at licenses.map.info (C:\Projekte\fire-dashboard\node_modules\@wbmnky\license-report-generator\dist\index.js:65:51)
    at Array.map (<anonymous>)
    at Promise.all.then.then.licenses (C:\Projekte\fire-dashboard\node_modules\@wbmnky\license-report-generator\dist\index.js:63:29)
(node:14476) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:14476) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

My repository for reproduction: https://github.com/erikhofer/fire-dashboard

**Comment by sebastianroming** [comment]
Hey @erikhofer, 

thanks for this issue, I'll look into it.

**Comment by sebastianroming** [comment]
I merged the PR (#15) and released 2.1.3.
Could you please verify if the addressed bug is fixed now?
